src/tools/genpolicy/src/mount_and_storage.rs

```diff
@@ -197,6 +197,13 @@ fn get_host_path_mount(
         }
     }
 
+    let access = match yaml_mount.readOnly {
+        Some(true) => {
+            debug!("setting read only access for host path mount");
```

Comment:
Do we need this?

Reply:
The debug print? It's not needed but it was useful for me while debugging and may be helpful in the future. [This](https://github.com/microsoft/kata-containers/blob/msft-main/src/tools/genpolicy/src/mount_and_storage.rs#L209) other similar one actually helped me understand the code path for the failing sample. Worth noting these prints only show if you run with `RUST_LOG=debug`